hack/vela-templates/cue/autoscale.cue

```diff
@@ -1,16 +1,18 @@
+import "strconv"
```

Comment:
I used CUE builtin package to solve the problem of not using double-quotes for int type parameter. To solve all double-quotes problems, please refer to the discussion https://github.com/oam-dev/kubevela/issues/585.

Reply:
As mentioned in #585, we prefer to fix that in Appfile impl. Though I personally not against having this workaround temporarily for autoscaler, please add a comment:
```
// Temporarily use strconv for type converting. This bug will be fixed in kubevela#585 
```